lib/gpu/lib/src/render_pass.dart

```diff
@@ -117,6 +180,15 @@ base class RenderTarget {
             colorAttachments: [colorAttachment],
             depthStencilAttachment: depthStencilAttachment);
 
+  _assertValid() {
+    for (final color in colorAttachments) {
+      color._assertValid();
+    }
+    if (depthStencilAttachment != null) {
+      depthStencilAttachment!._assertValid();
```

Comment:
But like I mention above I'd put all of this in an assert
```suggestion
      depthStencilAttachment?._assertValid();
```